nikola/data/themes/base/templates/author.tmpl

```diff
@@ -0,0 +1,43 @@
+## -*- coding: utf-8 -*-
+<%inherit file="list_post.tmpl"/>
+
+<%block name="extra_head">
+    ${parent.extra_head()}
+    %if len(translations) > 1 and generate_rss:
+        %for language in sorted(translations):
+            <link rel="alternate" type="application/rss+xml" type="application/rss+xml" title="RSS for ${kind} ${author} (${language})" href="${_link(kind + "_rss", author, language)}">
+        %endfor
+    %elif generate_rss:
+        <link rel="alternate" type="application/rss+xml" type="application/rss+xml" title="RSS for ${kind} ${author}" href="${_link(kind + "_rss", author)}">
+    %endif
+</%block>
+
+
+<%block name="content">
+<article class="authorpage">
+    <header>
+        <h1>${title|h}</h1>
+        %if description:
+        <p>${description}</p>
+        %endif
+        <div class="metadata">
+            %if len(translations) > 1 and generate_rss:
+                %for language in sorted(translations):
+                <p class="feedlink">
+                    <a href="${_link(kind + "_rss", author, language)}" hreflang="${language}" type="application/rss+xml">${messages('RSS feed', language)} (${language})</a>&nbsp;
+                </p>
+                %endfor
+            %elif generate_rss:
```

Comment:
I didn't add it because tag.tmpl does not (no atom reference). tagindex.tmpl does and authorindex.tmpl does too.